workshops/forms.py

```diff
@@ -71,3 +71,11 @@ class SearchForm(forms.Form):
     in_persons = forms.BooleanField(label='in persons',
                                     required=False,
                                     initial=True)
+
+class DebriefForm(forms.Form):
+    '''Represent general debrief form.'''
+
+    begin_date = forms.CharField(label='Begin date as YYYY-MM-DD',
+                           max_length=10)
+    end_date = forms.CharField(label='End date as YYYY-MD-DD',
```

Comment:
Little typo; should be "End date as YYYY-MM-DD". I'd also add a `min_length=10` parameter to help with validation. Or do we accept formats like "YYYY-MM"?

(We can go pro and use a custom validator https://docs.djangoproject.com/en/1.7/ref/validators/#writing-validators to make sure a valid date is provided. But I don't see it as a necessity.)